jenkins/ec2_bootstrap.sh

```diff
@@ -423,6 +423,24 @@ prepare_cowbuilder() {
     /bin/bash ./bootstrap.sh
 }
 
+install_datadog_agent() {
+    # Install the Datadog agent. The agent requires an API key which is expected
+    # to be present in the shell environment as DATADOG_AGENT_API_KEY. If the
+    # value is not present then inform stdout but do not sys exit.
```

Comment:
s/then inform stdout but do not exit/then complain and do not install the agent/